### ISSUE TITLE
Add batchnorm parameters: running_mean, running_var, and others

### ISSUE THREAD

**Issue by mostafaelhoushi** [sksq96/pytorch-summary#87]
Batchnorm has other parameters - in addition to `weight` and `bias` - that are not counted in torchsummary, such as `running_mean` and `running_var`:

https://pytorch.org/docs/stable/_modules/torch/nn/modules/batchnorm.html

**Comment by mostafaelhoushi** [comment]
I have created a pull request to address this issue:
https://github.com/sksq96/pytorch-summary/pull/88
